config/checkstyle_non_main_files_suppressions.xml

```diff
@@ -71,6 +71,16 @@
   <suppress id="lineLength"
          files="src[\\/]test[\\/]resources[\\/].*[\\/]checks[\\/]translation[\\/]Expected.*\.xml"/>
 
+  <!-- config attributes cannot be wraped -->
+  <suppress id="lineLength"
+         files="src[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck.java"/>
+  <suppress id="lineLength"
+         files="src[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck2.java"/>
+  <suppress id="lineLength"
+         files="src[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck3.java"/>
+  <suppress id="lineLength"
+         files="src[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheckOnDifferentTokens.java"/>
```

Comment:
https://checkstyle.org/config_filters.html#SuppressionFilter

use lines, `lines="2-5"/>` and let presume token property will be always one of top

or even better is update:
https://github.com/checkstyle/checkstyle/blob/13481f2c410e4944ecf5ab93ec49948a523a0c82/config/checkstyle_checks.xml#L214

to skip lines that start with "tokens="

Reply:
It was actually here:
https://github.com/checkstyle/checkstyle/blob/13481f2c410e4944ecf5ab93ec49948a523a0c82/config/checkstyle_non_main_files_checks.xml#L18-L24

Reply:
done now